public/app/plugins/datasource/graphite/specs/query_ctrl.jest.ts

```diff
@@ -0,0 +1,368 @@
+import { uiSegmentSrv } from 'app/core/services/segment_srv';
+// import { describe, beforeEach, it, sinon, expect, angularMocks } from 'test/lib/common';
+
+import gfunc from '../gfunc';
+// import helpers from 'test/specs/helpers';
+import { GraphiteQueryCtrl } from '../query_ctrl';
+
+describe('GraphiteQueryCtrl', () => {
+  let ctx = <any>{
+    datasource: {
+      metricFindQuery: jest.fn(() => Promise.resolve([])),
+      getFuncDefs: jest.fn(() => Promise.resolve(gfunc.getFuncDefs('1.0'))),
+      getFuncDef: gfunc.getFuncDef,
+      waitForFuncDefsLoaded: jest.fn(() => Promise.resolve(null)),
+      createFuncInstance: gfunc.createFuncInstance,
+    },
+    target: { target: 'aliasByNode(scaleToSeconds(test.prod.*,1),2)' },
+    panelCtrl: {
+      refresh: jest.fn(),
+    },
+  };
+
+  ctx.panelCtrl.panel = {
+    targets: [ctx.target],
+  };
+
+  // beforeEach(angularMocks.module('grafana.core'));
```

Comment:
Guess these comments should be removed. There are a few more at the top

Reply:
Thanks! Got a little bit too excited, I guess :)